### PR TITLE
Replacing chalk-printer with debug (fixes #108)

### LINKED ISSUE #108: Use debug instead of chalk-printer
1. Removed chalk-printer dependency.
 2. Install [debug](https://github.com/visionmedia/debug) as a dependency.
 3. Replace chalk-printer log calls with debug.
    - Create logic namespace using the directory path. For example: `src/server/services/scraper/repos.js` use `services:scraper:repos` (never include `src/xxx` or `index`.

### PR DESCRIPTION
**Pull request by pnkapadia6**
https://github.com/developersdo/opensource/issues/108

### CODE REVIEW

File: src/server/services/github/index.js [hunk 1]
```diff
@@ -50,6 +50,6 @@ module.exports = {
 }
 
 function logRateLimit(response) {
-  print.trace('Rate limit:', response.data.rateLimit)
+  debug('DD - Rate limit:', response.data.rateLimit)
```
Comment (rmariuzzo) [inline src/server/services/github/index.js]: @pnkapadia6 what does the `DD -` stands for?

File: src/server/services/scraper/users.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const print = require('chalk-printer')
+const debug = require('debug')('services:scrapper:users')
```
Comment (rmariuzzo) [inline src/server/services/scraper/users.js]: Use scraper instead of scrapper. :-)

File: src/server/services/github/index.js (hunk 1 above)
Comment (pnkapadia6) [reply src/server/services/github/index.js]: Sending another PR with those changes

Comment (rmariuzzo) [reply src/server/services/github/index.js]: @pnkapadia6 that's fixed, don't worry. You did the hard work already.

File: src/server/services/scraper/users.js (hunk 2 above)
Comment (rmariuzzo) [reply src/server/services/scraper/users.js]: I took care of that.

**Pull request merged**